inflation/main.py

```diff
@@ -0,0 +1,22 @@
+from fastapi import FastAPI
+import logging
+from inflation.dataset.crawl import Crawler
+
+
+logging.basicConfig(level=logging.DEBUG)
+logger = logging.getLogger(__name__)
+
+app = FastAPI()
+crawler = Crawler()
+
+
+def fetch_inflation_data():
+    url = "https://docs.google.com/spreadsheets/d/1mZSqW_X_KuQQdGhH7oAn3-MATyfePLlYYmOgE-mB_9Q/edit#gid=0"
+    records = crawler.parse_excel_to_link_dataset(url)
+    crawler.crawl(records, "/inflation-resources/data/")
+
```

Comment:
can you add another **endpoint** to list all the files exist here as well as the line numbers?

for instance, the new endpoint can return:

`{"2022-02-13.jsonl": 150, "2022-02-13.jsonl": 155}` 

we can understand our data size. also for the next iteration, we should push the files to the cloud storage bucket. I'll let you know about it more.